api/queries_repo.go

```diff
@@ -1243,23 +1243,26 @@ func relevantProjects(client *Client, repo ghrepo.Interface) ([]RepoProject, []P
 
 	g, _ := errgroup.WithContext(context.Background())
 
-	g.Go(func() error {
-		var err error
-		repoProjects, err = RepoProjects(client, repo)
-		if err != nil {
-			err = fmt.Errorf("error fetching repo projects (classic): %w", err)
-		}
-		return err
-	})
-	g.Go(func() error {
-		var err error
-		orgProjects, err = OrganizationProjects(client, repo)
-		if err != nil && !strings.Contains(err.Error(), errorResolvingOrganization) {
-			err = fmt.Errorf("error fetching organization projects (classic): %w", err)
+	if includeProjects {
+		g.Go(func() error {
+			var err error
+			repoProjects, err = RepoProjects(client, repo)
+			if err != nil {
+				err = fmt.Errorf("error fetching repo projects (classic): %w", err)
+			}
 			return err
-		}
-		return nil
-	})
+		})
+		g.Go(func() error {
+			var err error
+			orgProjects, err = OrganizationProjects(client, repo)
+			if err != nil && !strings.Contains(err.Error(), errorResolvingOrganization) {
+				err = fmt.Errorf("error fetching organization projects (classic): %w", err)
+				return err
+			}
+			return nil
+		})
+	}
+
```

Comment:
It's not clear to me what `includeProjects` is doing, here. It _seems_ like it should be the switch for whether or not to query v1 projects, though it isn't named as such nor does it seem to be operating as such a conditional, here. I'd expect `OrganizationProjects` to always be called, and `RepoProjects` to get called conditionally. Maybe I'm missing something about how the featureDetector works?